### ISSUE TITLE
Cannot SSH to some devices

### ISSUE THREAD

**Issue by erdanilevicius** [raphamorim/rio#719]
I have decided to try out some different terminal emulators and found this one, It looks like everything with this emulator fits for me, but unfortunately, I am not able to login to some devices via SSH or Telnet. With majority of the remote machines I have tried everything is fine. But when trying to connect to a machine that is not linux I get "connection closed", meanwhile connecting to same devices via different emulators is possible.

**Comment by raphamorim** [comment]
Hey @erdanilevicius thanks for the issue, whenever you use Rio via SSH what's the TERM env? Could it be connected to this https://raphamorim.io/rio/docs/frequently-asked-questions/#i-get-errors-about-the-terminal-being-unknown-or-opening-the-terminal-failing-or-functional-keys-like-arrow-keys-dont-work ?

**Comment by erdanilevicius** [comment]
locally and in the device server that I want to SSH to other devices my TERM env is xterm-256color. It was the first thing I was looking into, because of experience with kitty.

**Comment by raphamorim** [comment]
I see! Whenever you do SSH on Rio, the server TERM should match with the host otherwise can lead to weird behaviours. Like kitty also ask about it https://sw.kovidgoyal.net/kitty/faq/#i-get-errors-about-the-terminal-being-unknown-or-opening-the-terminal-failing-or-functional-keys-like-arrow-keys-don-t-work

**Comment by erdanilevicius** [comment]
> the server TERM should match with the host otherwise can lead to weird behaviours

Yep, unfortunately, in my case it matches, I'm unable to do certain things, like mentioned in this thread. Is it like with kitty, that if using different terminfo than the terminal emulator comes with(in this case using xterm-256color instead of rio), can break some of the functionality?

**Comment by raphamorim** [comment]
> if using different terminfo than the terminal emulator comes with(in this case using xterm-256color instead of rio), can break some of the functionality?

yes,

but you can run `TERM=xterm-256color ssh` every time that you connect via ssh or install rio terminfo on your server: https://raphamorim.io/rio/docs/install/terminfo

**Comment by erdanilevicius** [comment]
> but you can run TERM=xterm-256color ssh every time that you connect via ssh 

Unfortunately, I have that one set as alias in my zshrc to pass it, but it doesn't fix the original issue for me, all other terminal emulators work as intended.

**Comment by shtrophic** [comment]
same issue here. `$TERM` is both `xterm-256color` on both (OpenBSD) host and client, and the ssh connection immediately terminates.

**Comment by pinpox** [comment]
Is there any drawback to just always setting `xterm-256color` for TERM? I've seen other terminals do this (e.g. foot terminal), they just always set xterm-256color for compatibility, even though it is a different terminal emulator.
